src/vivarium/framework/results/strategies.py

```diff
@@ -99,3 +100,69 @@ def _bin_data(data: pd.Series) -> pd.Series:
             return pd.cut(data, bins, labels=labels, **cut_kwargs)
 
         super().__init__(target, binned_column, _bin_data, is_vectorized=True)
+
+
+class FormattingStrategy(abc.ABC):
+    """Base interface for results formatting strategies.
+
+    Formatting strategies turn aggregated results into a final output
+    format for results production.
+
+    """
+
+    def __init__(self, measure: str, **additional_keys: Dict[str, str]):
+        """
+        Parameters
+        ----------
+        measure
+            The measure this strategy is formatted to produce.
+        additional_keys
+            Additional labels to attach to the formatted data.
+
+        """
+        self._measure = measure
+        self._additional_keys = additional_keys
+
+    @staticmethod  # NOTE: Do not override.  This method is final.
+    def _broadcast_aggregates(aggregate_data: pd.Series) -> pd.DataFrame:
+        """Broadcasts aggregate data over unobserved categories.
```

Comment:
Test for this to follow.  I need a production strategy to produce aggregates first.